workday/13.0/modules/ROOT/pages/workday-connector-upgrade-migrate.adoc

```diff
@@ -8,245 +8,38 @@ Upgrade Anypoint Connector for Workday (Workday Connector) to version 13.x.x.
-                ns0#Last_Name: vars.testData.employeeLastName,
-              }
-            }
-            },
-            ns0#Contact_Data: {
-                ns0#Email_Address_Data @(ns0#"Delete": "false" , ns0#"Do_Not_Replace_All": "false") : {
-                    ns0#Email_Address : vars.testData.employeeEmailAddress,
-                    ns0#Usage_Data @(ns0#"Public": "false"): {
-                        ns0#Type_Data @(ns0#"Primary": "true") : {
-                            ns0#Type_Reference @(ns0# "Descriptor": "Type_Reference"): {
-                                ns0#ID @(ns0#"type": "Communication_Usage_Type_ID "): vars.testData.communicationUsageTypeID
-                            }
-                      }
-                    }
-                }
-            }
-          }
-        },
-        ns0#Organization_Reference @(ns0# "Descriptor": "Organization_Reference"): {
-            ns0#ID @(ns0#"type": "Organization_Reference_ID"): vars.testData.organizationReferenceID
-        },
-        ns0#Position_Reference @(ns0# "Descriptor": "Position_Reference"): {
-            ns0#ID @(ns0#"type": "WID"): payload.ns0#Create_Position_Response.ns0#Position_Reference.ns0#ID
-        },
-        ns0#Hire_Date: vars.testData.currentDate,
-        ns0#Hire_Employee_Event_Data: {
-            ns0#Employee_Type_Reference @(ns0# "Descriptor": "Hire_Reference"): {
-                ns0#ID @(ns0#"type": "Employee_Type_ID "): vars.testData.employeeTypeID
-            },
-            ns0#Position_Details: {
-                ns0#Job_Profile_Reference @(ns0# "Descriptor": "Hire_Reference"): {
-                    ns0#ID @(ns0#"type": "Job_Profile_ID "): vars.testData.jobProfileID
-                },
-                ns0#Location_Reference @(ns0# "Descriptor": "Hire_Reference"): {
-                    ns0#ID @(ns0#"type": "Location_ID"): vars.testData.locationID
-                },
-                ns0#Position_Time_Type_Reference @(ns0# "Descriptor": "Hire_Reference"): {
-                    ns0#ID @(ns0#"type": "Position_Time_Type_ID"): vars.testData.positionTimeTypeID
-                }
-            }
-        }
-      }
-    }
-}]]></ee:set-payload>
-            </ee:message>
-        </ee:transform>
-        <workday:staffing operation="Hire_Employee" doc:name="Hire Employee" config-ref="Workday-Config" />
-    </flow>
-    <flow name="listener-flow" >
-        <workday:modified-objects-listener objectType="WORKERS" doc:name="On Modified Objects Listener" config-ref="Workday-Config">
-            <scheduling-strategy >
-                <fixed-frequency />
-            </scheduling-strategy>
-        </workday:modified-objects-listener>
-        <logger level="INFO" doc:name="Logger - On Modified Objects" message='#[payload]'/>
-    </flow>
-    </mule>
-----
+* Upgraded the Workday API to 35.0
+* Added the *Moments* operation, which enables you to invoke the Workday Moments Web Service
```

Comment:
I searched for "workday moments web service" and "workday moments." I found mostly "Web Service: Moments" and two verions, "of the Moment" and one other I can't recall. It was a mixed bag. 
```suggestion
* Added the *Moments* operation, which enables you to invoke the Workday Moments web service
```